Apps/Sandcastle/gallery/CZML Path.html

```diff
@@ -0,0 +1,1007 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+  <meta name="description" content="CZML Path">
+  <meta name="cesium-sandcastle-labels" content="CZML">
+  <title>Cesium Demo</title>
+  <script type="text/javascript" src="../Sandcastle-header.js"></script>
+  <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.9/require.js"></script>
+  <script type="text/javascript">
+  require.config({
+    baseUrl : '../../../Source',
+    waitSeconds : 60
+  });
+  </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+@import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+  "use strict";
+  //Sandcastle_Begin
+  var czml = [
+    {
+      "id" : "document",
+      "name" : "CZML Path",
+      "version" : "1.0"
+    }, {
+      "id" : "path with GPS flight data",
+      "availability" : "2012-08-04T10:00:00Z/2012-08-04T17:00:00Z",
+      "position" : {
+        "epoch" : "2012-08-04T10:00:00Z",
+        "cartographicRadians" : [
```

Comment:
> It's from one of my friends, who's a hang glider pilot, so no credit necessary (although I could add in a hat tip if you like).

Please confirm we have permission to use it and by all means include their name and a link in the CZML description.

> I think lat/long are switched on this, although for example purposes I suppose that doesn't matter?

We should fix it since I thought this was an odd place for the path.

With the right editor or a small script, it should be quick.